test/db/tools/rz_bin

```diff
@@ -678,19 +678,19 @@ vaddr      paddr      type   name
 
 4 relocations
 4899
-Version symbols section '.gnu.version' contains 5 entries:
- Addr: 0x08048246  Offset: 0x00000246  Link: 5 (.dynsym)
+Version symbols has 5 entries:
+ Addr: 0x08048246  Offset: 0x00000246
   0x00000000: 0 (*local*)
   0x00000001: 2 (GLIBC_2.0)
   0x00000002: 0 (*local*)
   0x00000003: 2 (GLIBC_2.0)
   0x00000004: 1 (*global*)
 
 
-Version need section '.gnu.version_r' contains 1 entries:
- Addr: 0x08048250  Offset: 0x00000250  Link to section: 6 (.dynstr)
-  0x00000000: Version: 1  File: libc.so.6  Cnt: 1
-  0x00000010:   Name: GLIBC_2.0  Flags: none Version: 2
+Version need has 1 entries:
+ Addr: 0x08048250  Offset: 0x00000250
+  0x00000250: Version: 1  File: libc.so.6  Cnt: 1
+  0x00000260:   Name: GLIBC_2.0  Flags: none Version: 2
```

Comment:
why these changes of offset?

Reply:
The old offset was from the .gnu.version_r. I switch to the offset in the file.

The reason is that it simplify the implementation.

Reply:
In the example the first offset is the verneed struct and the second offset is a vernaux struct.